jax/lib/xla_bridge.py

```diff
@@ -50,6 +50,12 @@ def _check_jaxlib_version():
 from jaxlib import xla_client
 from jaxlib import xla_data_pb2
 
+# Workaround for older jaxlib versions. Remove after a jaxlib release.
```

Comment:
Leave a TODO here so it's easier to spot?

Reply:
Done.